### PR TITLE
Ug restructure features

### LINKED ISSUE #158: UG: Restructure Features Section
Currently the features segment is just a long list of commands. It should be split into smaller groups based on functionality (such as leaves, performance, etc.) accompanied with a good description of the feature, its intended uses, and so on.

### PR DESCRIPTION
**Pull request by kevinchangjk**
**Resolves #158** 

Made some changes in line with the aforementioned task.

- Reorganised the `docs/images` folder
- Restructured the features segment of the user guide
    - Grouped into four main functions
    - Added additional features section
    - Added additional headings for parts I think needs its own description
- Added logo and introduction
- Added guide to the user guide segment (content is empty)

There are a lot of places where the content has to be updated. 

I have marked the ones that I added specifically, as "**[Insert ...]**". Please provide feedback on those if you have suggestions, and for the relevant ones, please fill them in yourself in your future PRs.

### CODE REVIEW

**Comment by ngshijun** [comment]
LGTM

File: docs/UserGuide.md [hunk 1]
```diff
@@ -3,30 +3,68 @@ layout: page
...
+    - [Employee Performance History](#employee-performance-history)
+  - [Managing Departments](#managing-departments)
+    - [View details of a department](#view-details-of-a-department--viewdepartment)
+  - [Additional Features](#additional-features)
+    - [Exiting the program](#exiting-the-program--exit)
+    - [Clearing the data](#clearing-the-data--clear)
+    - [Saving the data](#saving-the-data)
+    - [Editing the data file](#editing-the-data-file)
+- [FAQ](#faq)
+- [Command summary](#command-summary)
```
Comment (chao890) [inline docs/UserGuide.md:50]: Very nice.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -40,49 +78,70 @@ Coydir is a desktop app to manage the employee details within a company, optimiz
...
-    e.g. if the command specifies `help 123`, it will be interpreted as `help`.
+- Extraneous parameters for commands that do not take in parameters (such as `help`, `list`, `exit` and `clear`) will be ignored.<br>
+  e.g. if the command specifies `help 123`, it will be interpreted as `help`.
 
 :warning: **Integer input values cannot be too large**: any values greater than $2^{32}-1$ is not supported.
 
-</div>
+### Managing Employee Profiles
+
+What's Human Resources without the Human aspect? (*P.S. It's not "Resources"*)
```
Comment (chao890) [inline docs/UserGuide.md:134]: That is hilarious!

**Review by chao890 — APPROVED** [review]
Good job documentation IC.

**Pull request merged**